components/Modal/RaffleRewardModal.js

```diff
@@ -83,6 +85,24 @@ const RaffleRewardModal = () => {
 							</>
 						)}
 					</p>
+					<div className="mt-2">
```

Comment:
ini kalau misalnya `data?.msg.twitterShareContent` nya ga ada tetep muncul ya `share`nya?

Reply:
muncul gann, content sharenya dari backend

Reply:
ooh dibikin ilang aja ga share button-nya kalau ga ada dari BE?